src/web/controllers/admin/event_admin_controller.py

```diff
@@ -672,37 +673,72 @@ def download_players_as_vcf(
             },
         )
 
+    datasheet_columns = [
+        'last_name',
+        'first_name',
+        'yob',
+        'mail',
+        'phone',
+        'gender',
+        'fide_id',
+        'tournament',
+        'federation',
+        'club',
+        'St',
+        'S',
+        'Ra',
+        'R',
+        'Bl',
+        'B',
+    ]
+    
+    @staticmethod
+    def get_players_datasheet_extra_columns() -> dict[int, ExtraColumn]:
+        """Returns the extra data columns added by the plugins"""
+        per_plugin_columns = plugin_manager.hook.get_extra_players_datasheet_columns()
+        extra_columns = {}
+        for plugin_columns in per_plugin_columns:
+            for extra_column in plugin_columns:
+                try:
+                    index = EventAdminController.datasheet_columns.index(extra_column.at)
```

Comment:
if it's a static method accessing a class constant of method, it should be a class method
```suggestion
    @classmethod
    def get_players_datasheet_extra_columns(cls) -> dict[int, ExtraColumn]:
        """Returns the extra data columns added by the plugins"""
        per_plugin_columns = plugin_manager.hook.get_extra_players_datasheet_columns()
        extra_columns = {}
        for plugin_columns in per_plugin_columns:
            for extra_column in plugin_columns:
                try:
                    index = cls.datasheet_columns.index(extra_column.at)
```